proto/redpanda/api/auth/v1/authorization.proto

```diff
@@ -22,6 +22,7 @@ enum API {
   API_REDPANDA_CONNECT = 4;
   API_KAFKA_CONNECT = 5;
   API_CONSOLE = 6;
+  API_REDPANDA_MCP_SERVER = 7;
```

Comment:
Is `API_REDPANDA_MCP_SERVER` the best name for this? I'd omit `REDPANDA` or use `API_REDPANDA_CONNECT_MCP_SERVER`.

Also worth to consider re-using `API_REDPANDA_CONNECT` here since it's literally the same API that we are talking too. Not sure it makes sense to separate these out since we could create MCP servers using the pipelines API as well I believe? No strong opinion here either

Reply:
Since this is used for authz, I think it makes sense to split it out and have fine grained resources